profiler/README.md

```diff
@@ -15,3 +52,84 @@ It should be considered experimental and is completely unsupported.
+[otel.javaagent 2021-09-28 18:17:04:244 +0000] [main] INFO <snip> -              splunk.profiler.keep-files : false
+[otel.javaagent 2021-09-28 18:17:04:245 +0000] [main] INFO <snip> -           splunk.profiler.logs-endpoint : null
+[otel.javaagent 2021-09-28 18:17:04:245 +0000] [main] INFO <snip> -             otel.exporter.otlp.endpoint : http://collector:4317
+[otel.javaagent 2021-09-28 18:17:04:245 +0000] [main] INFO <snip> -            splunk.profiler.tlab.enabled : false
+[otel.javaagent 2021-09-28 18:17:04:246 +0000] [main] INFO <snip> -   splunk.profiler.period.jdk.threaddump : null
+[otel.javaagent 2021-09-28 18:17:04:246 +0000] [main] INFO <snip> - -----------------------
+```
+
+### What about this escape hatch?
+
+If the escape hatch becomes active, it will log with `com.splunk.opentelemetry.profiler.RecordingEscapeHatch`
+(you can grep for this in the logs). You may also look for `"** THIS WILL RESULT IN LOSS OF PROFILING DATA **"`
+as a big hint that things are not well.
+
+You may need to free up some disk space and/or give the JVM more resources.
+
+### What if I'm on an unsupported JVM?
+
+If your JVM does not support JFR, this is detected by the profiler at startup. It will log
+"Java Flight Recorder (JFR) is not available in this JVM. Profiling is disabled." at `WARN`,
+and you can grep for this in the logs.
+
+### Why is the OTLP/logs exporter complaining?
+
+This can happen for a number of reasons, but is generally caused by the collector being misconfigured.
+If the logs cannot be exported to the collector, the profiling data will not be available in
+the UI.
+
+Things to check:
+
+* Look at the values of the agent's configuration: `splunk.profiler.logs-endpoint` and `otel.exporter.otlp.endpoint`. Hint: they are logged
+at startup (see above).
+* Verify that a collector is actually running at that endpoint and that the
+application host/container can resolve any hostnames and actually connect to the given OTLP port (default: 4317)
+* Make sure you are running the [Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector)
+and that the version is 0.33.1 or greater. Other collector distributions may not be able to route the log
+data containing profiles correctly.
+* Make sure that the collector is configured correctly to handle profiling data. By default, the
+[Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector) handles this, but
+a custom configuration might have overridden some settings. Make sure that an OTLP _receiver_ is configured in the collector
+and that an exporter is configured for `splunk_hec` export. Ensure that the `token` and `endpoint` fields
+are [correctly configured](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/splunkhecreceiver#configuration).
+Lastly, double check that the logs _pipeline_ is configured to use the OTLP receiver and `splunk_hec` exporter.
```

Comment:
* Make sure that the collector is configured correctly to handle profiling data. By default, the [Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector) handles profiling data, but custom configuration might override profiling settings.
* * Make sure that an OTLP _receiver_ is configured in the collector and that an exporter is configured for `splunk_hec` export.
* * Make sure that the `token` and `endpoint` fields are [correctly configured](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/splunkhecreceiver#configuration).
* * Make sure that the logs _pipeline_ is configured to use the OTLP receiver and `splunk_hec` exporter.